mlflow/models/evaluation/lift_curve.py

```diff
@@ -0,0 +1,153 @@
+import matplotlib.pyplot as plt
```

Comment:
@WeichenXu123 Nice! Can we add docstrings to the methods in this module indicating that they are taken from `scikit-plot`? Also, since this module defines both `lift_curve` and `cumulative_gain_curve`, shall we use a different name?